### PR TITLE
Reorganize VNC roles

### PR DESCRIPTION
**Pull request by calvinmclean**
This PR combines the three VNC roles (`realvncserver`, `novnc`, and `guacamole`) into one role. The `main.yml` tasks file will check if GUI is available, then include a tasks file for each `realvncserver`, `novnc`, and `guacamole`. 

- `realvncserver.yml`
  - Create directories for VNC
  - Install dependencies and VNC Server
  - Copy over configuration files and templates
  - Change desktop background (might make this its own task file so that related tasks since it is not directly related to realvnc)
  - Set VNC password and start VNC Server on `:1`

- `novnc.yml`
  - Install NoVNC
  - Start VNC Server on `:2` and novnc websocket

- `guacamole.yml`
  - Add VNC config for Guacamole
  - Start VNC Server on `:5` for Guacamole

### CODE REVIEW

File: ansible/roles/atmo-vnc/tasks/main.yml [hunk 1]
```diff
@@ -27,6 +28,12 @@
...
-  include: install.yml
+- block:
+    - include: realvnc.yml
+      when: SETUP_REALVNC_SERVER is defined and SETUP_REALVNC_SERVER == true
+
+    - include: guacamole.yml
+      when: SETUP_GUACAMOLE is defined and SETUP_GUACAMOLE == true
+
+    - include: novnc.yml
   when: has_gui
```
Comment (c-mart) [inline ansible/roles/atmo-vnc/tasks/main.yml]: Indentation is off here. Also, I think we should have a `SETUP_NOVNC` variable to make the NoVNC stuff optional.

File: ansible/roles/atmo-vnc/tasks/realvnc.yml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ---
 
 - name: set environment variables for legacy systems
-  set_fact: ansible_python_interpreter=/usr/bin/python
+  set_fact:
+    ansible_python_interpreter: /usr/bin/python
```
Comment (c-mart) [inline ansible/roles/atmo-vnc/tasks/realvnc.yml]: How many times do we need this in a role? I think you also have it in `tasks/main.yml` and maybe 1-2 other places.

**Review by c-mart — COMMENTED** [review]
Looks good to me at a glance. I say merge it into your fork if it produces the desired result under the various configurations (only guacamole, only NoVNC, and both).

File: ansible/roles/atmo-vnc/tasks/realvnc.yml (hunk 2 above)
Comment (calvinmclean) [reply ansible/roles/atmo-vnc/tasks/realvnc.yml]: It's old code, I wasn't sure if I could trash it. I'm pretty sure it's not necessary, but I'll have to double check that.

Comment (c-mart) [reply ansible/roles/atmo-vnc/tasks/realvnc.yml]: I would think once near the top of `tasks/main.yml` will suffice, then it'll run once each time the role is called

**Pull request merged**